example/src/test/java/com/vivy/springdoc/scoped/startertest/StarterApiTests.java

```diff
@@ -45,4 +59,26 @@ public void apiDocIsGenerated() {
         assertThat(apiDocs.getBody()).contains("paths\":{\"/test\":{\"get");
     }
 
+    @Test
+    public void testThatApiDocFileIsCreatedByNameAndLocationSpecifiedInPropertiesFiles() {
+        String apiDocFileName = environment.getProperty("openapi-boot-starter.swagger-ui.api-docs.file.name");
+        String apiDocFilePath = environment.getProperty("openapi-boot-starter.swagger-ui.api-docs.file.path");
```

Comment:
you don't need to create a separate properties for path and name - it could be one property, which you could use as `Path` and just take a name directly from it

Reply:
done